presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -1153,4 +1154,19 @@ public void testComplexCast()
                         "SELECT count(DISTINCT a), CAST(max(b) AS VARCHAR) FROM t",
                 "VALUES (1, '0 00:00:01.000')");
     }
+
+    @Test
+    public void testCSECompilation()
```

Comment:
Is there a reason this is in AbstractTestDistributedQueries rather than AbstractTestQueries?

Reply:
No specific reason, moved it to AbstractTestQueries.